Release Candidate 1.0.3 fails to build for iOS Device

Simply create a little iOS app, add Realm and **build for a device** and you get the following linker errors;

**Note** a workaround is to turn off Linking - the default project settings for iOS are _Link Framework SDKs Only_

```
		/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++  -framework UIKit -framework Foundation -framework QuartzCore -framework CoreGraphics -framework CloudKit -weak_framework CFNetwork /Library/Frameworks/Xamarin.iOS.framework/Versions/10.4.0.96/SDKs/MonoTouch.iphoneos.sdk/usr/lib/libmonosgen-2.0.dylib /Library/Frameworks/Xamarin.iOS.framework/Versions/10.4.0.96/SDKs/MonoTouch.iphoneos.sdk/usr/lib/libxamarin-debug.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/registrar.arm64.o /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/main.arm64.o /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/Try103LinkDevice.exe.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/Xamarin.iOS.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/mscorlib.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/System.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/System.Core.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/Mono.Dynamic.Interpreter.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/Realm.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/DotNetCross.Memory.Unsafe.dll.arm64.dylib /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/obj/iPhone/Debug/device-builds/iphone7.1-10.2.1/mtouch-cache/Remotion.Linq.dll.arm64.dylib -force_load /Library/Frameworks/Xamarin.iOS.framework/Versions/10.4.0.96/SDKs/MonoTouch.iphoneos.sdk/usr/lib/libapp.a -Wl,-pie -arch arm64 -miphoneos-version-min=10.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS10.2.sdk -lz -o /Users/andydent/temp/Try103LinkDevice/Try103LinkDevice/bin/iPhone/Debug/device-builds/iphone7.1-10.2.1/Try103LinkDevice.app/Try103LinkDevice -liconv -L/Users/andydent/temp/Try103LinkDevice/packages/Realm.1.0.3/build/../native/ios/universal -lrealm-wrappers -lz -u _xamarin_timezone_get_data -u _xamarin_GetFolderPath -u _xamarin_log -u _query_binary_equal -u _query_binary_not_equal -u _query_string_contains -u _query_string_starts_with -u _query_string_ends_with -u _query_string_equal -u _query_string_not_equal -u _query_bool_equal -u _query_bool_not_equal -u _query_int_equal -u _query_int_not_equal -u _query_int_less -u _query_int_less_equal -u _query_int_greater -u _query_int_greater_equal -u _query_long_equal -u _query_long_not_equal -u _query_long_less -u _query_long_less_equal -u _query_long_greater -u _query_long_greater_equal -u _query_float_equal -u _query_float_not_equal -u _query_float_less -u _query_float_less_equal -u _query_float_greater -u _query_float_greater_equal -u _query_double_equal -u _query_double_not_equal -u _query_double_less -u _query_double_less_equal -u _query_double_greater -u _query_double_greater_equal -u _query_timestamp_ticks_equal -u _query_timestamp_ticks_not_equal -u _query_timestamp_ticks_less -u _query_timestamp_ticks_less_equal -u _query_timestamp_ticks_greater -u _query_timestamp_ticks_greater_equal -u _query_object_equal -u _query_null_equal -u _query_null_not_equal -u _query_find -u _query_find_next -u _query_get_column_index -u _query_not -u _query_group_begin -u _query_group_end -u _query_or -u _query_destroy -u _query_count -u _query_create_results -u _query_create_sorted_results -u _shared_realm_open -u _shared_realm_set_managed_state_handle -u _shared_realm_get_managed_state_handle -u _shared_realm_destroy -u _shared_realm_close_realm -u _shared_realm_begin_transaction -u _shared_realm_commit_transaction -u _shared_realm_cancel_transaction -u _shared_realm_get_table -u _shared_realm_add_observed_object -u _shared_realm_remove_observed_object -u _table_add_empty_object -u _table_unbind -u _table_create_results -u _object_for_string_primarykey -u _object_for_int_primarykey -u _object_for_null_primarykey -u _register_notify_realm_changed -u _delete_pointer -u _register_notify_realm_object_changed -u _realm_install_gchandle_deleter -u _list_add -u _list_insert -u _list_erase -u _list_clear -u _list_get -u _list_find -u _list_size -u _list_destroy -u _list_move -u _results_is_same_internal_results -u _results_destroy -u _results_get_row -u _results_count -u _results_get_query -u _object_get_is_valid -u _object_destroy -u _object_set_timestamp_ticks -u _object_get_timestamp_ticks -u _object_get_nullable_timestamp_ticks -u _object_set_string -u _object_set_string_unique -u _object_get_string -u _object_set_link -u _object_get_link -u _object_clear_link -u _object_get_list -u _object_set_null -u _object_set_null_unique -u _object_set_bool -u _object_get_bool -u _object_get_nullable_bool -u _object_set_int64 -u _object_set_int64_unique -u _object_get_int64 -u _object_get_nullable_int64 -u _object_set_float -u _object_get_float -u _object_get_nullable_float -u _object_set_double -u _object_get_double -u _object_get_nullable_double -u _object_set_binary -u _object_get_binary -u _object_equals_object -u _object_get_backlinks -u _collection_destroy_notificationtoken -u _realm_install_eventloop_callbacks -u _UIApplicationMain -u _xamarin_get_block_descriptor -u _mono_pmip
		Undefined symbols for architecture arm64:
		  "_SSLClose", referenced from:
		      realm::util::network::ssl::Stream::ssl_shutdown(std::__1::error_code&, realm::util::network::Service::Want&) in librealm-wrappers.a(network_ssl-iphoneos.o)
		      realm::util::network::ssl::Stream::do_ssl_handshake() in librealm-wrappers.a(network_ssl-iphoneos.o)
		      realm::util::network::ssl::Stream::do_ssl_shutdown() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLCopyPeerTrust", referenced from:
		      realm::util::network::ssl::Stream::verify_peer() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLCreateContext", referenced from:
		      realm::util::network::ssl::Stream::ssl_init() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLGetSessionState", referenced from:
		      realm::util::network::ssl::Stream::ssl_shutdown(std::__1::error_code&, realm::util::network::Service::Want&) in librealm-wrappers.a(network_ssl-iphoneos.o)
		      realm::util::network::ssl::Stream::ssl_read(char*, unsigned long, std::__1::error_code&, realm::util::network::Service::Want&) in librealm-wrappers.a(network_ssl-iphoneos.o)
		      realm::util::network::ssl::Stream::do_ssl_shutdown() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLHandshake", referenced from:
		      realm::util::network::ssl::Stream::do_ssl_handshake() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLRead", referenced from:
		      realm::util::network::ssl::Stream::ssl_read(char*, unsigned long, std::__1::error_code&, realm::util::network::Service::Want&) in librealm-wrappers.a(network_ssl-iphoneos.o)
		      realm::util::network::ssl::Stream::do_ssl_read(char*, unsigned long) in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLSetConnection", referenced from:
		      realm::util::network::ssl::Stream::ssl_init() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLSetIOFuncs", referenced from:
		      realm::util::network::ssl::Stream::ssl_init() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLSetPeerDomainName", referenced from:
		      realm::util::network::ssl::Stream::ssl_set_check_host(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::error_code&) in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLSetProtocolVersionMin", referenced from:
		      realm::util::network::ssl::Stream::ssl_init() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLSetSessionOption", referenced from:
		      realm::util::network::ssl::Stream::ssl_init() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SSLWrite", referenced from:
		      realm::util::network::ssl::Stream::do_ssl_write(char const*, unsigned long) in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_SecItemAdd", referenced from:
		      realm::keychain::metadata_realm_encryption_key() in librealm-wrappers.a(keychain_helper.o)
		  "_SecItemCopyMatching", referenced from:
		      realm::keychain::metadata_realm_encryption_key() in librealm-wrappers.a(keychain_helper.o)
		  "_SecTrustEvaluate", referenced from:
		      realm::util::network::ssl::Stream::verify_peer() in librealm-wrappers.a(network_ssl-iphoneos.o)
		  "_kSecAttrAccessGroup", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecAttrAccessible", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecAttrAccessibleAlways", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecAttrAccount", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecAttrService", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecClass", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecClassGenericPassword", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecReturnData", referenced from:
		      realm::keychain::build_search_dictionary(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, realm::util::Optional<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >) in librealm-wrappers.a(keychain_helper.o)
		  "_kSecValueData", referenced from:
		      realm::keychain::metadata_realm_encryption_key() in librealm-wrappers.a(keychain_helper.o)
		ld: symbol(s) not found for architecture arm64
		clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/NuGet/Realm.Database/Realm.Database.targets
+++ b/NuGet/Realm.Database/Realm.Database.targets
@@ -20,6 +20,7 @@
       <SmartLink>True</SmartLink>
       <IsCxx>True</IsCxx>
       <LinkerFlags>-lz</LinkerFlags>
+      <Frameworks>Security</Frameworks>
     </NativeReference>
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == 'MonoAndroid'">

--- a/Platform.XamarinIOS/Tests.XamarinIOS/Tests.XamarinIOS.csproj
+++ b/Platform.XamarinIOS/Tests.XamarinIOS/Tests.XamarinIOS.csproj
@@ -150,6 +150,7 @@
       <SmartLink>True</SmartLink>
       <IsCxx>True</IsCxx>
       <LinkerFlags>-lz</LinkerFlags>
+      <Frameworks>Security</Frameworks>
     </NativeReference>
   </ItemGroup>
   <Import Project="..\..\packages\StyleCop.MSBuild.4.7.54.0\build\StyleCop.MSBuild.Targets" Condition="Exists('..\..\packages\StyleCop.MSBuild.4.7.54.0\build\StyleCop.MSBuild.Targets')" />

